src/extensions/windows/Microsoft.DotNet.UpgradeAssistant.Extensions.Windows/UWPtoWinAppSDKUpgrade/UWPToWinUIHelpers.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Diagnostics;
+
+namespace Microsoft.DotNet.UpgradeAssistant.Extensions.Windows.UWPtoWinAppSDKUpgrade
+{
+    internal static class UWPToWinUIHelpers
+    {
+        public static IEnumerable<string> GetAllImportedNamespaces(this SyntaxNodeAnalysisContext context)
+        {
+            return context.Node.Ancestors().OfType<CompilationUnitSyntax>().First().DescendantNodes().OfType<UsingDirectiveSyntax>()
```

Comment:
What about no CompilationUnitSyntax found?

Also, why `SyntaxNodeAnalysisContext`? `SyntaxNode` would probably be better. This is also C# specific, so it would be good to convey that in some way